sections/all_map_locations.tex

```diff
@@ -115,15 +120,15 @@ \subsection*{Symbols on the Map}
 
   \phantom{j}\textbf{Pandora's Box}\\
   \framedimage[\linewidth]{\map_locations/pandoras_box.jpg}\\
-}
 
-\vspace*{\fill}
-
-\vspace{7.4em}
+  \bigskip
 
-\note{10}{
-  The effects of Fields that allow you to \textbf{Search} for Spells or Artifacts, or where you must spend resources to use the Field's effect, are \textbf{always optional}.
-  You must always place a Black Cube on such Visitable Fields even if you choose not to use that Field's effect.
+  \begin{figure}[H]
```

Comment:
```suggestion
  \begin{figure}[h]
```
Try experimenting with this, or `!h`. Don't use `H`, as it's not native LaTeX, and it's breaking po4a. Use it only as a last resort.